cookbooks/bcpc/attributes/graphite.rb

```diff
@@ -25,3 +25,5 @@
 default['bcpc']['graphite']['carbon']['cache']['MAX_UPDATES_PER_SECOND'] = 3000
 
 default['bcpc']['graphite']['ip'] = node['bcpc']['management']['vip']
+default['bcpc']['graphite']['use_whitelist'] = False
```

Comment:
I don't think this will compile. it should be `false` in ruby or a string `"False"`  since this is basically what you want inside the python config file.